templates/options.erb

```diff
@@ -0,0 +1,15 @@
+Dpkg::Options {
+<%- @_options.each_pair do |config, value| 
+  if %w(force_confdef force_confold force_confnew force_confmiss).include?(config) then
+    if value.is_a?(TrueClass) || value.is_a?(FalseClass) then 
+      if value then -%>
```

Comment:
I could add individual checks in the puppet manifest like 

```puppet
validate_bool($options['force_confdef'])
validate_bool($options['force_confold'])
validate_bool($options['force_confnew'])
validate_bool($options['force_confmiss'])
```

And remove the check in the erb if you would prefer